geotrek/feedback/filters.py

```diff
@@ -1,17 +1,31 @@
-from geotrek.zoning.filters import ZoningFilterSet
+from django.utils.translation import gettext_lazy as _
+from django_filters import MultipleChoiceFilter
 from mapentity.filters import MapEntityFilterSet
+
+from geotrek.zoning.filters import ZoningFilterSet
+
 from .models import Report
 
 
 class ReportFilterSet(ZoningFilterSet, MapEntityFilterSet):
+    year_insert = MultipleChoiceFilter(
+        label=_("Creation year"),
+        field_name='date_insert__year',
+        choices=lambda: Report.objects.year_insert_choices()
+    )
+    year_update = MultipleChoiceFilter(
+        label=_("Update year"),
+        field_name='date_update__year',
+        choices=lambda: Report.objects.year_update_choices()
+    )
 
     class Meta(MapEntityFilterSet.Meta):
         model = Report
-        fields = ['activity', 'email', 'category', 'status', 'problem_magnitude', 'assigned_user']
+        fields = ['activity', 'category', 'status', 'problem_magnitude', 'assigned_user']
 
 
-class ReportNoEmailFilterSet(ZoningFilterSet, MapEntityFilterSet):
+class ReportEmailFilterSet(ReportFilterSet):
 
     class Meta(MapEntityFilterSet.Meta):
         model = Report
-        fields = ['activity', 'category', 'status', 'problem_magnitude', 'assigned_user']
+        fields = ['activity', 'email', 'category', 'status', 'problem_magnitude', 'assigned_user']
```

Comment:
why change email in a filter dedicated PR ?

Reply:
I reorganized these two filtersets to
- have a more logical inheritance
- not need to duplicate my code by adding the two new filters in both FilterSets

## Before :

`ReportFilterSet` -> filters on 10 fields

`ReportNoEmailFilterSet` ->  filters on 9 fields because it does include 'email' filter, and **does not inherit** from `ReportFilterSet` 

## After :

`ReportFilterSet` -> filters on 12 fields
`ReportEmailFilterSet` -> filters on same 12 fields + 'email' = 13 fields and **does inherit** from ReportFilterSet